### DISCUSSION TITLE
1.0 Preview 2 Release is LIVE!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1551] [Announcements]
## 1.0 Preview 2 Release of Windows App SDK
 
We've just released version 1.0 Preview 2 of the Windows App SDK. This is the first version that adds Windows 11 styles to WinUI 3! And single-project MSIX project templates are now included with the VSIX, so you can easily create a new packaged project that uses one single project! There are a few known issues around single-project MSIX documented in the release notes. This is a great opportunity to try out what's new and provide feedback before the 1.0 Stable (GA) release!
 
- If you're building an app with the Windows App SDK for the first time, follow the [getting started instructions here.](https://docs.microsoft.com/windows/apps/project-reunion/get-started-with-project-reunion)

- [Install the C# or C++ extensions](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment) for the 1.0 Preview 2 release.

 
To see everything that's new and changed, see the full [release notes here](https://docs.microsoft.com/windows/apps/windows-app-sdk/preview-channel#version-10-preview-2-100-preview2).
 
## What's next
To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://github.com/microsoft/ProjectReunion/blob/main/docs/roadmap.md), which gets updated regularly. 

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.0.0-preview2'>1.0 Preview 2 Release is LIVE!</a>.</em>

**Comment by ghost1372** [comment]
Hi @andrewleader Thank you, This release was super fast and a surprise I am very happy😅
it seems that mica is not available Is that right?

**Reply by andrewleader** [reply]
Thanks! Yep this is the release that has the Win11 styles you were asking for!

You can ask the XAML team about Mica, I believe it is not available in this release, try asking them here: https://github.com/microsoft/microsoft-ui-xaml/issues

**Reply by andrewleader** [reply]
Thanks! Yep this is the release that has the Win11 styles you were asking for!

You can ask the XAML team about Mica, I believe it is not available in this release, try asking them here: https://github.com/microsoft/microsoft-ui-xaml/issues

**Reply by nickrandolph** [reply]
I think some of the acrylic/mica implementations were leaked into preview 1 and 2 - see https://github.com/microsoft/microsoft-ui-xaml/issues/6037
It's not obvious how you use the micacontroller but you can hack around with it and get the mica background to render in your app. It's not clear how it should be used to set the background of your app/controls

**Comment by riverar** [comment]
For folks looking for the runtime redistributable package that contains the MSIX files and runtime installers https://aka.ms/windowsappsdk/1.0-preview2/msix-installer

**Reply by nickrandolph** [reply]
Where is this link documented? and why isn't it included in the release notes :-)

**Reply by andrewleader** [reply]
This link is documented in the deployment guide for unpackaged apps: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/deploy-unpackaged-apps#prerequisites

> Download the latest [Windows App SDK installer and MSIX packages](https://aka.ms/windowsappsdk/1.0-preview2/msix-installer).

Including that link in the release notes would likely be useful though, I'll share that feedback with the team!

**Comment by lhak** [comment]
Are unpackaged winui 3 applications already supposed to work? I can get a lot further now when starting as an unpackaged app but see a crash (System.Runtime.InteropServices.COMException) when setting the SelectedItem property of a NavigationView.

**Reply by andrewleader** [reply]
Hey @lhak, unpackaged WinUI 3 is getting closer but still not ready to try out in Preview 2. We're constantly making more steps towards having unpackaged WinUI 3 working every day (it's our top priority right now with daily internal updates 😜). We'll definitely announce when unpackaged WinUI 3 is available to try!

**Reply by shelllet** [reply]
🎆

**Comment by younky-yang** [comment]
Just curious, I am not sure how to make the C# app to build with dotNet native compiler or it is not supported on WinUI although it build as the UWP app.
